docs/source/resources/frequently_asked_questions.ipynb

```diff
@@ -229,23 +230,38 @@
+    "                         new_dataframe_name=\"sessions\",\n",
     "                         index=\"session_id\",\n",
     "                         make_time_index=\"session_start\",\n",
-    "                         additional_variables=[\"session_start\"])\n",
+    "                         additional_columns=[\"session_start\"])\n",
     "es.plot()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The type for `session_id` is now `Id` in the `transactions` entity, and `Index` in the new entity, `sessions`. This is the case because when we normalize the entity, we create a new relationship between the `transactions` and `sessions`. There is a one to many relationship between the parent entity, `sessions`, and child entity, `transactions`.\n",
+    "The `session_id` now has the sematic tag `foreign_key` in the `transactions` DataFrame, and `index` in the new DataFrame, `sessions`. This is the case because when we normalize the DataFrame, we create a new relationship between the `transactions` and `sessions`. There is a one to many relationship between the parent DataFrame, `sessions`, and child DataFrame, `transactions`.\n",
     "\n",
-    "Therefore, `session_id` has type `Id` in `transactions` because it represents an `Index` in another entity. There would be a similar effect if we added another entity using `add_dataframe` and `add_relationship`. \n",
+    "Therefore, `session_id` has the semantic tag `foreign_key` in `transactions` because it represents an `index` in another DataFrame. There would be a similar effect if we added another DataFrame using `add_dataframe` and `add_relationship`. \n",
     "\n",
-    "In addition, when we created the new entity, we specified a `time_index` which was the variable (column) `session_start`. This changed the type of `session_start` to `datetime_time_index` in the new `sessions` entity because it now represents a time_index."
+    "In addition, when we created the new DataFrame, we set `session_start` as the `time_index`. This added the semantic tag `time_index` to the `session_start` column in the new `sessions` DataFrame because it now represents a `time_index`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How do I update a column's description or metadata?\n",
+    "\n",
+    "You can directly update the attributes of the column's description or metadata. However, you must specifically use the column returned by `DataFrame.ww.columns['col_name']`, **not** `DataFrame.ww['col_name']`. The column from `DataFrame.ww.columns['col_name']` is still associated with the EntitySet and propagates any updates to the column attributes. In contrast, the column from `DataFrame.ww['col_name']` is a new Series not associated with the EntitySet and does not propagate any updates to the column attributes. As an example, this is how you can update a column's description or metadata:\n",
```

Comment:
From [this comment](https://github.com/alteryx/featuretools/issues/1578#issuecomment-895449176), I understood that we want to directly update the attributes on the column and that we want users to use `df.ww.columns[col_name]` instead of `df.ww[col_name]`. I didn't see anything about typing info or schemas.

Both approaches return a column, so I'm not exactly sure how replacing it with typing info would clarify?

Reply:
Nvm, I see how `df.ww.columns[col_name]` returns a column schema. I'll update to clarify.

Reply:
Perhaps it'd make sense to use column schema instead of typing info?

Reply:
No worries! It's clear that this concept trips up pretty much every single woodwork user, so we probably need to be excessive about where and when we warn about this.

Also, column schema makes much more sense than typing info and says in line with how we talk about this elsewhere!